hyper/http20/connection.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, host, port=None, secure=None, window_manager=None, enable_pus
         #: network. This is used as a performance optimisation. Increase buffer
         #: size to improve performance: decrease it to conserve memory.
         #: Defaults to 64kB.
-        self.network_buffer_size = 65536
+        self.network_buffer_size = 2**24
```

Comment:
This is not a flow control value, it's just an in-memory buffer to store TCP data. We can leave this as-is for now.